### PR TITLE
fix crash wrt notifications

### LINKED ISSUE #720: GlideApp...getI() must not be called from main thread
got two crashes wrt notifications on nexus4/android 8.1/oreo

the crashes were assertions from the Glide module; saying "You must call this method on a background thread":
https://gist.githubusercontent.com/r10s/28ef216fad00e1e1fd467cac71daa804/raw/84bc8797f71e3aeda23083197302ff157b6a3378/gistfile1.txt

tracing back the log, it's probably [SingleRecipientNotificationBuilder.setThread()](https://github.com/deltachat/deltachat-android/blob/master/src/org/thoughtcrime/securesms/notifications/SingleRecipientNotificationBuilder.java#L75) which calls GlideApp...get(), probably from the main thread and not from a background thread.

wondering why the app does not crash all the time.

### PR DESCRIPTION
**Pull request by r10s**
closes #720

### CODE REVIEW

File: src/org/thoughtcrime/securesms/notifications/MessageNotifier.java [hunk 1]
```diff
@@ -174,7 +180,7 @@ public static void updateNotification(@NonNull  Context context,
                                         int       threadId,
                                         boolean   signal)
   {
-    boolean    isVisible  = visibleThread == threadId;
+    boolean    isVisible  = visibleChatId == threadId;
```
Comment (angelo-fuchs) [inline src/org/thoughtcrime/securesms/notifications/MessageNotifier.java]: should changename of threadId here too.

**Comment by angelo-fuchs** [comment]
I just tested it and the following issue bugs me:
When I already have Delta open and there are notifications. When I open a chat the notification stays until I leave that chat again.

**Comment by adbenitez** [comment]
yes! I was preparing screenshots for a bug report for this issue!

**Comment by adbenitez** [comment]
in my case not only until I go back but until I go into the chat again?!

**Comment by r10s** [comment]
> When I already have Delta open and there are notifications. When I open a chat the notification stays until I leave that chat again.

yip, i think there are still lots of tiny issues wrt notificatrions, however, for now merging this pr so that the crashes stop :)

**Comment by angelo-fuchs** [comment]
@adbenitez I opened #725 for further treatment

**Pull request merged**